ajax.php

```diff
@@ -34,7 +34,7 @@
      * Type: HTTP POST
      *
      * Parameters:
-     * 	- content_id
+     *     - content_id
```

Comment:
I see what happened with this one. Looking at the previous codechecker
report, it found a tab character, which Moodle doesn't allow. It then
replaced the tab character with four spaces as per the Moodle convention.
It didn't change the others, as indent spacing is not a code requirement in
comment blocks. Odd affect. I'll make a manual change and resubmit.
mike

On Thu, Mar 31, 2016 at 2:57 AM Juho Jaakkola <notifications@github.com>
wrote:

> In ajax.php
> <https://github.com/h5p/h5p-moodle-plugin/pull/42#discussion_r58008048>:
>
> > @@ -34,7 +34,7 @@
> >       * Type: HTTP POST
> >       *
> >       * Parameters:
> > -     * 	- content_id
> > +     *     - content_id
>
> This seems weird. Is the indentation deliberately different from the other
> items in the list?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/h5p/h5p-moodle-plugin/pull/42/files/74840234bb2d141cd0c7c4ead1e6f695f62e7888#r58008048>
>
-- 

Mike Churchward | Executive Director | POET
mike.churchward@poetgroup.org
1.519.301.2287
http://poetgroup.org